### ISSUE TITLE
Cumulative yield plot

### ISSUE THREAD

**Issue by devindrown** [wdecoster/NanoPlot#7]
There appears to be a large discrepancy between the cumulative yield plot y-axis scale and the summary output. For a sample run, the Nanostats output (consistent with other programs) show ~1.4Gbase

```
$ cat NanoStats.txt
Number of reads:        243724
Total bases:    1435869352

```
However, the plot yield vs. time maxes out at 0.012 gigabases
![fullcumulativeyieldplot](https://user-images.githubusercontent.com/4990488/28442656-e848b6be-6d5d-11e7-885f-8a51cb5c6424.png)

**Comment by wdecoster** [comment]
Hmmm I'll need to have a look at this, thanks for letting me know.

**Comment by wdecoster** [comment]
This turned out to be a quite ridiculous small mistake. It's fixed in NanoPlot-0.14.4.

(The actual change is in nanoplotter-0.13.1 but NanoPlot depends on the latest version of nanoplotter. So updating NanoPlot suffices.)

Cheers,
W
